package.json

```diff
@@ -224,10 +224,10 @@
     "jest-resolve": "^27.0.6",
     "jest-watch-typeahead": "^0.4.2",
     "nock": "^11.7.0",
-    "node-sass": "^4.14.1",
     "prop-types": "^15.7.2",
     "redux-devtools-extension": "^2.13.8",
     "redux-mock-store": "^1.5.3",
+    "sass": "^1.55.0",
```

Comment:
Any reason why this is being changed in this PR?

Reply:
Req'd for Node 16

Reply:
Thought as much, but believed that was being tackled in a separate PR. Just noticed this PR has two commits. Cool!